packages/ui/src/lib/checkbox/Checkbox.svelte

```diff
@@ -33,20 +33,20 @@ function onClick(
       class:cursor-pointer={!disabled}
       class:cursor-not-allowed={disabled}>
       {#if disabled}
-        <Fa size="1.1x" icon={faSquare} class="text-[var(--pd-input-checkbox-disabled)]" />
+        <Fa size="1.3x" icon={faSquare} class="text-[var(--pd-input-checkbox-disabled)]" />
       {:else if indeterminate}
         <Fa
-          size="1.1x"
+          size="1.3x"
           icon={faMinusSquare}
           class="text-[var(--pd-input-checkbox-indeterminate)] hover:text-[var(--pd-input-checkbox-focused-indeterminate)]" />
       {:else if checked}
         <Fa
-          size="1.1x"
+          size="1.3x"
           icon={faCheckSquare}
           class="text-[var(--pd-input-checkbox-checked)] hover:text-[var(--pd-input-checkbox-focused-checked)]" />
       {:else}
         <Fa
```

Comment:
I feel like for consistency we should have a dedicated component, allowing us to have a single places to change this value

Reply:
Do you mean component or just constant? I set a constant in a new commit.